ci/SetupDevEnv/src/SetupDevEnv.jl

```diff
@@ -0,0 +1,101 @@
+module SetupDevEnv
+
+using TOML
+using Pkg
+
+"""
+    extract_env_vars_from_git_message!()
+
+Parse the commit message, if set via variable `CI_COMMIT_MESSAGE` and set custom urls.
+"""
+function extract_env_vars_from_git_message!()
+    if haskey(ENV, "CI_COMMIT_MESSAGE")
+        println("Found env variable CI_COMMIT_MESSAGE")
```

Comment:
Consider using `@info` instead of `println`, because this works better with the logging module (cf. [here](https://docs.julialang.org/en/v1/stdlib/Logging/)).